Dockerfile.sdk

```diff
@@ -134,10 +134,8 @@ RUN cd install && \
     export VERSION=`cat /workspace/TRITON_VERSION` && \
     tar zcf /workspace/v$VERSION.clients.tar.gz *
 
-# For CI testing need to install a test script.
-COPY qa/L0_sdk/test.sh /tmp/test.sh
-COPY qa/L0_sdk/grpc_test.cc /tmp/grpc_test.cc
-COPY qa/L0_sdk/http_test.cc /tmp/http_test.cc
+# For CI testing need to copy over L0_sdk test.
+COPY qa/L0_sdk L0_sdk
```

Comment:
So just going to have a /workspace/L0_sdk directory in the SDK image? That doesn't seem great. Also, the L0_sdk test will need to be updated for this new location.

Reply:
Updated the SDK test on GitLab and created a PR for this. We want to avoid using /tmp or at least making it configurable. For this case I believe it does not make sense to add an additional param to dockerfile for just 1 test. 
How about /workspace/qa/L0_sdk?

Reply:
I agree. /workspace/qa/L0_sdk is a good idea.